.github/workflows/cdn.yml

```diff
@@ -11,79 +11,13 @@ jobs:
-          echo ::set-output name=PATCH::$(echo $GITHUB_REF | cut -d / -f 3 | cut -d '.' -f 3)
+        run: echo ::set-output name=TAG::$(echo $GITHUB_REF | cut -d / -f 3)
 
-      - name: Set up Cloud SDK
-        uses: google-github-actions/setup-gcloud@master
+      - name: Trigger storage
+        uses: aurelien-baudet/workflow-dispatch@v2
         with:
-          project_id: ${{ secrets.G_PROJECT_PRODUCTION }}
-          service_account_key: ${{ secrets.G_CREDENTIAL_PRODUCTION }}
-          export_default_credentials: true
-
-      - name: Checkout repository
-        uses: actions/checkout@master
-
-      - uses: actions/cache@v2
-        id: cache-deps
-        with:
-          path: '_dev/node_modules'
-          key: ${{ runner.os }}-modules-${{ hashFiles('_dev/package-lock.json') }}
-
-      - uses: actions/setup-node@v1
-        with:
-          node-version: 14
-          cache: npm
-          cache-dependency-path: '_dev/package-lock.json'
-
-      - name: Install dependencies
-        working-directory: _dev
-        if: steps.cache-deps.outputs.cache-hit != 'true'
-        run: yarn install
-
-      - name: Build prerelease
-        working-directory: _dev
-        if: "github.event.release.prerelease"
-        run: npm run build
-        env:
-          VUE_APP_CDN_URL: https://storage.googleapis.com/psxmarketing-cdn/${{ steps.get_tag.outputs.MAJOR }}.${{ steps.get_tag.outputs.MINOR }}.${{ steps.get_tag.outputs.PATCH }}/
-
-      - name: Build
-        working-directory: _dev
-        if: "!github.event.release.prerelease"
-        run: npm run build
-        env:
-          VUE_APP_CDN_URL: https://storage.googleapis.com/psxmarketing-cdn/${{ steps.get_tag.outputs.MAJOR }}.x.x/
-
-      - name: Delete old CDN
-        continue-on-error: true
-        if: "!github.event.release.prerelease"
-        run: |
-          gsutil rm -r gs://psxmarketing-cdn/${MAJOR}.x.x
-          gsutil rm -r gs://psxmarketing-cdn/${MAJOR}.${MINOR}.x
-        env:
-          MAJOR: ${{ steps.get_tag.outputs.MAJOR }}
-          MINOR: ${{ steps.get_tag.outputs.MINOR }}
-
-      - name: Prerelease - Push to CDN
-        if: "github.event.release.prerelease"
-        run: |
-          gsutil cp -r ./views gs://psxmarketing-cdn/${MAJOR}.${MINOR}.${PATCH}
-        env:
-          MAJOR: ${{ steps.get_tag.outputs.MAJOR }}
-          MINOR: ${{ steps.get_tag.outputs.MINOR }}
-          PATCH: ${{ steps.get_tag.outputs.PATCH }}
-
-      - name: Release - Push to CDN
-        if: "!github.event.release.prerelease"
-        run: |
-          gsutil cp -r ./views gs://psxmarketing-cdn/${MAJOR}.x.x
-          gsutil cp -r ./views gs://psxmarketing-cdn/${MAJOR}.${MINOR}.x
-        env:
-          MAJOR: ${{ steps.get_tag.outputs.MAJOR }}
-          MINOR: ${{ steps.get_tag.outputs.MINOR }}
-          PATCH: ${{ steps.get_tag.outputs.PATCH }}
-
+          workflow: metrics-storage.yml
```

Comment:
Need to double check this file name.

I wonder if we can have the customizable variable always at the top of our workflow file to make sure we never forget them

Reply:
Thanks, wrong workflow here.
Good idea, I created an env on top of our config !